src/main/java/com/lob/model/Letter.java

```diff
@@ -175,9 +178,11 @@ public String getAddressPlacement() {
     }
 
     public boolean isReturnEnvelope() {
-        return returnEnvelope;
+        return true;

```

Comment:
@mwj8410 is this supposed to be `return returnEnvelope != null;`? returning default true here is not what we want i think

Reply:
absolutely correct. I will update immediately